pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -84,6 +84,18 @@ def work(self, response_dict=None):
         # get pokemon data
         pokemon_data = response['wild_pokemon']['pokemon_data'] if 'wild_pokemon' in response else response['pokemon_data']
         pokemon = Pokemon(self.pokemon_list, pokemon_data)
+        
+        # get all available pokeballs
+        items_stock = self.bot.current_inventory()
+        current_ball = ITEM_POKEBALL
+        num_ball = 0
+        while current_ball < ITEM_ULTRABALL:
```

Comment:
If the goal is to just count the number of poke balls (of any type), this seems like it has a side effect in that it is changing which ball it would be using. 

Perhaps @bricesd or @amal have a better idea how we should handle doing this check. It seems like we do a lot of these kinds of things in this file.